### PR TITLE
Simplify launch prod peers

### PR DESCRIPTION
**Pull request by lvh**
This builds on top of #95. It does not add the tests that I wanted to add, but it does make it a little bit cleaner. I stopped here, because I think the way forward is to merge the launcher code into one namespace, as #95 does with the single run.sh.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,65 +1,23 @@
...
+
+# Cache the dependencies in a Docker fs layer.
 RUN lein deps
-COPY . /usr/src/desdemona
-RUN mv "$(lein uberjar | sed -n 's/^Created \(.*standalone\.jar\)/\1/p')" /srv/desdemona.jar
 
-ADD script/run_peers.sh /etc/service/onyx_peer/run
-ADD script/run_aeron.sh /etc/service/aeron/run
+RUN lein uberjar
+RUN mv target/desdemona-*-standalone.jar /srv/desdemona.jar
```
Comment (derwolfe) [inline Dockerfile]: Just out of curiosity - is it important that the uberjar is formed as a separate layer from the the move step? I'd lean towards consolidating the `lein uberjar` and `mv ...` into a single `RUN` command.

Comment (lvh) [inline Dockerfile]: I'm not sure; I suppose not :)

**Comment by derwolfe** [comment]
Modulo my single comment, which is entirely optional, I think this is ready to go. :-).

**Comment by lvh** [comment]
Thanks @derwolfe!

**Pull request merged**